integrationtest/wal_rollback_test.go

```diff
@@ -286,14 +265,3 @@ func checkRoleBindingExists(k8sClient kubernetes.Interface, listOptions metav1.L
 		return len(bindings.Items) > 0, nil
 	}
 }
-
-func checkServiceAccountExists(k8sClient kubernetes.Interface, listOptions metav1.ListOptions) (bool, error) {
```

Comment:
Maybe we should be checking whether the service account exists or not in the tests? I don't remember why it was excluded.

Reply:
I guess it wasn't included because the test is designed to not have permission to generate the service account? In any case, I've added the function back and added an assertion that it isn't ever created.

Reply:
Oh yep, you're right 👍